app/tests/profiles_tests/test_views.py

```diff
@@ -181,3 +182,60 @@ def test_organization_update(self, client):
         )
 
         assert org1.title not in response.content.decode()
+
+
+@pytest.mark.django_db
+class TestAccountCreation(TestCase):
```

Comment:
Please write pytest style tests. Subclassing `TestCase` is not necessary and we should avoid the weird `.assertWhatever` methods.

Reply:
think this is fixed - tests are now pytest style, I think